pkg/cli/admin/inspect/proxy.go

```diff
@@ -0,0 +1,61 @@
+package inspect
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+
+	"k8s.io/cli-runtime/pkg/resource"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+var proxyRegex = regexp.MustCompile(`(https?://)([^:]+):([^@]+)@`)
+
+type proxyList struct {
+	*configv1.ProxyList
+}
+
+func (c *proxyList) addItem(obj interface{}) error {
```

Comment:
Is this method used anywhere? Is there an interface proxyList needs to implement?

Reply:
It needs the implementation of this function to satisfy `listAccessor` interface in https://github.com/openshift/oc/blob/89781e0b8f29052e3350b74b677f3c69eea096f1/pkg/cli/admin/inspect/resource.go#L128 to be populated into the list https://github.com/openshift/oc/blob/89781e0b8f29052e3350b74b677f3c69eea096f1/pkg/cli/admin/inspect/resource.go#L179

Reply:
`var _ listAccessor = &proxyList{}` will do then

Reply:
This https://github.com/openshift/oc/pull/1929/files#diff-6f6b5698b5c7ebfc01c64967180a4d2e5f2f248509dae99f7d0d07e13018c366R151-R153 ensures that `proxyList` implements `listAccessor` but in any case, I've added your suggestion above.